src/client/user/User.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { browserHistory } from 'react-router';
+import loginStyles from './Users.css';
+
+
+export default class User extends Component {
```

Comment:
I think this is a fairly specialised component, just for lobby. I might make more sense to have it in the lobby directory